docs/sphinx/contributing/modules.rst

```diff
@@ -4,13 +4,13 @@
    Distributed under the Boost Software License, Version 1.0. (See accompanying
    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-.. _libs__example:
+.. _module_structure:
 
-==============
-Example module
-==============
+================
+Module structure
+================
 
-This is an example module used to explain the structure of an |hpx| module.
+This section explains the structure of an |hpx| module.
 
 The tool `create_library_skeleton.py
 <https://github.com/STEllAR-GROUP/hpx/blob/master/libs/create_library_skeleton.py>`_
```

Comment:
Shouldn't we at least add a sentence or two here? Now that we remove the example there is no 'go by example' anymore...

Reply:
What do you have in mind? I agree in principle, but `create_library_skeleton.py` essentially gives one a minimal example module.

Reply:
Well, perhaps just a sentence explaining that running this script (how - any command line arguments?) generates a minimal set of files resembling an HPX module...

Reply:
Updated it to explain that and a bit of `add_hpx_module`.

Reply:
Thanks!